src/NuGet.Core/NuGet.ProjectModel/PackageSpecWriter.cs

```diff
@@ -235,6 +235,11 @@ private static void WriteNuGetAuditProperties(IObjectWriter writer, RestoreAudit
             SetValueIfNotNull(writer, "auditLevel", auditProperties.AuditLevel);
             SetValueIfNotNull(writer, "auditMode", auditProperties.AuditMode);
 
+            if (auditProperties.SuppressedAdvisories.Any())
```

Comment:
```suggestion
            if (auditProperties.SuppressedAdvisories.Count > 0)
```

Never use LINQ when it's not needed. It causes allocations and has higher CPU cost than checking properties that types already provide.

Reply:
Fixed